### PR TITLE
remove coreos-operator dependancy

### PR DESCRIPTION
**Pull request by krasi-georgiev**
TODO:
- [x] change the openshift configs
- [x] update all README.md files

### CODE REVIEW

**Comment by objectiser** [comment]
@krasi-georgiev Wasn't sure if you meant me to try it out yet, or still work in progress - but gave it a quick try. Looks good - got the results in prom, although for some reason the jaeger UI just spins, nothing to do with your PR.

Before doing the final PR, I think it would be good to change the label "frontend" to something more meaningful - I think that was just a name prom-operator used in their example. It just needs to be something metric related I guess.

**Comment by krasi-georgiev** [comment]
Great I will continue with openshift and will replace the label.

**Comment by krasi-georgiev** [comment]
all done , I tested both just to make sure.

The jaeger UI worked once and than it gets stuck at the spinning circle.  
I can see in the google dev tools that the data is there but the ui doesn't render it and I only found this issue which seems to be the solution.
https://github.com/jaegertracing/jaeger-ui/issues/115

File: OpenShift.md [hunk 1]
```diff
@@ -4,14 +4,9 @@ This example will use [minishift](https://docs.openshift.org/latest/minishift/ge
...
+minishift start
 ```
 
-You can use a more [recent version](https://github.com/openshift/origin/releases) if available.
-
-NOTE: The alpha version is currently required, as the previous stable version does not include Kubernetes RBAC authorizaion
-support.
-
 When fully started, then launch the dashboard using the link displayed following `The server is accessible via web console at:
 `. Log in using the credentials `developer/developer` and navigate to the `New Project` overview page.
```
Comment (objectiser) [inline OpenShift.md]: The next line contains "Kubernetes", could you change this to "OpenShift" - looks like I missed that (copy/paste) :)

**Comment by objectiser** [comment]
@krasi-georgiev Great job - just a few minor comments.

However the main issue is that the metrics scraped by prometheus, vs prom-operator, are lacking labels for "service", "pod" and "namespace".

There is an 'instance' label - but that doesn't seem to be specific to the individual pod - so if we have multiple instances of a particular service, there is no way to determine which instance actually executed the request,

Even more important is 'service' (and 'namespace' to a less extent).

Any ideas of how these labels could be added in, without the application/service itself including them into the metrics?

**Pull request merged**